context/build-utils/webpack.common.js

```diff
@@ -14,19 +13,20 @@ const config = {
     splitChunks: {
       cacheGroups: {
         commons: {
-          test: RegExp('/node_modules/'),
+          test: /\/node_modules\//,
           name: 'vendors',
           chunks: 'initial',
         },
       },
     },
   },
   resolve: {
-    extensions: ['.js', '.jsx', '.css', '.woff', '.woff2', '.svg', '.yaml', '.yml', '.json'],
+    extensions: ['.js', '.jsx', '.ts', '.tsx', '.css', '.woff', '.woff2', '.svg', '.yaml', '.yml', '.json'],
     fallback: {
       // Now necessary because webpack 5 doesn't include these polyfills by default
       timers: require.resolve('timers-browserify'),
       stream: require.resolve('stream-browserify'),
+      buffer: require.resolve('buffer'),
```

Comment:
What required introducing the polyfill?

Reply:
Same xml2js library as before started throwing errors during docker build until I included this. I added this to the related ticket so it's removed alongside the other ones.